extensions/ql-vscode/src/variant-analysis/store/repo-states-data-types.ts

```diff
@@ -0,0 +1,12 @@
+export interface VariantAnalysisScannedRepositoryStateData {
+  repositoryId: number;
+  downloadStatus: VariantAnalysisScannedRepositoryDownloadData;
+  downloadPercentage?: number;
+}
+
+export enum VariantAnalysisScannedRepositoryDownloadData {
```

Comment:
Nice! This looks good. It's good these "data types" files don't import any other files, and that shows we don't accidentally depend on a domain model.